km/km.h

```diff
@@ -183,6 +191,7 @@ typedef struct km_vcpu {
    uint64_t dr_regs[4];   // remember the addresses we are watching and have written into
                           // the processor's debugging facilities in DR0 - DR3.
    gdb_vcpu_state_t gdb_vcpu_state;   // gdb's per thread (vcpu) state.
+   km_snap_state_t snap_state;
```

Comment:
Could you add a statement in km_vcpu_init() to set snap_state to its initial value.  It currently defaults to 0 (SNAP_STATE_RUNNING).

Reply:
ok